docs/server/source/appendices/tendermint-integration.rst

```diff
@@ -1,26 +0,0 @@
-######################
```

Comment:
You will also have to remove the reference to this page from the file that generates the table of contents for the Appendices, i.e. `docs/server/source/appendices/index.rst`

Currently, when I go to the `/docs/server/` directory and do `make html`, I get the error message:
```text
Warning, treated as error:
/home/troy/repos/bigchaindb/docs/server/source/appendices/index.rst:4:toctree contains reference to nonexisting document 'appendices/tendermint-integration'
Makefile:56: recipe for target 'html' failed
make: *** [html] Error 2
```

Reply:
Fixed